spec/unit/pops/serialization/to_from_hr_spec.rb

```diff
@@ -296,10 +296,14 @@ def parse(string)
           expect(val2).to be_a(Types::PObjectTypeExtension)
           expect(val2).to eql(val)
         end
+
+        it 'with POjbectTypeExtension type being converted' do
+          val = { 'ObjectExtension' => type.create(34) }
+          expect { write(val) }.not_to raise_error
```

Comment:
I'm not familiar with this code, but generally I try to avoid expecting something to never raise, as the test might pass when it shouldn't. But if the method doesn't return anything, then sometimes it's unavoidable. Is there something we could expect, like `expect(write(val)).to be_a(Types::PObjectTypeExtension)`?

Reply:
Certainly! For a little more context, the `convert` function was what was causing the error. I used the existing method for consistency, but you make a good point that I think more specific tests are more important than consistency. Updated!